modules/bitgo/test/v2/unit/coins/utxo/recovery/mock.ts

```diff
@@ -61,26 +63,55 @@ export class MockRecoveryProvider implements RecoveryProvider {
   }
 }
 export class MockCrossChainRecoveryProvider<TNumber extends number | bigint> implements RecoveryProvider {
+  private addressVersion: 'cashaddr' | 'base58';
+  private addressFormat: utxolib.addressFormat.AddressFormat;
   constructor(
     public coin: AbstractUtxoCoin,
     public unspents: Unspent<TNumber>[],
     public tx: utxolib.bitgo.UtxoTransaction<TNumber>
-  ) {}
+  ) {
+    // this is how blockchair will return the data, as a cashaddr for BCH like coins
+    // BSV supports cashaddr, but at the time of writing the SDK does not support cashaddr for bsv
```

Comment:
It was too big a lift to add cashadr support for BSV in this PR as well (and out of scope), hence why it is excluded in the following lines.